### ISSUE TITLE
Devemos ou não renomear o repositório de "casos-de-sucesso-na-adocao-de-animais" para "casos-de-sucesso"

### ISSUE THREAD

**Issue by fititnt** [pokemongovet/casos-de-sucesso-na-adocao-de-animais#8]
Este repositório, hoje, tem o nome de "casos-de-sucesso-na-adocao-de-animais", e, talvez, seja interessante renomear para "casos-de-sucesso".
### Motivos positivos
- É um pouco mais genérico. Embora adoção de animais seja o foco, PokemonGo está ajudando muito mais do que isso
- Pode atrair mais ajuda além de amantes de animais
- É mais curto
### Motivos negativos
- Irrita desenvolvedores
- Pode não ser claro para desenvolvedores

Gostaria da opinião de vocês. Caso não haja objeções fortes de fazermos essa mudança, eu recomendaria que ela fosse feita.

Issue relacionado https://github.com/pokemongovet/pokemongo.vet.br/issues/6

**Comment by anabastos** [comment]
Eu acho que reduzir apenas pra "casos-de-sucesso" não deixa tão claro a o que repositório se refere mas nada realmente drástico.

**Comment by evansantos** [comment]
@fititnt concordo com a @anabastos mesmo estando debaixo da organização, ele meio que faz o nome virar algo que não está tão claro
